meilisearch-release-changelog/v1.2.0.md

```diff
@@ -0,0 +1,112 @@
+# v1.2.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.2.0 improve search relevancy and performance but also introduces new filter operators like `IS NULL` and `IS EMPTY`!
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+<!-- The following line should NOT be put in the official release changelogs, and the link should be added as soon as possible during the pre-release -->
+📖 Meilisearch's official documentation will be ready for the official v1.2.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Delete documents using filtering
+
+TBD
+
+Done by @dureuill and @irevoire in #3550.
+
+## New filter operators: `IS EMPTY` and `IS NULL`
+
+During the search, you can use [filtering](https://www.meilisearch.com/docs/reference/api/search#filter) and retrieve documents containing `null` or empty fields.
+
+For example with the following set of documents:
+
+```json
+[{
+    "id": 0,
+    "color": []
+},
+{
+    "id": 1,
+    "color": null
+},
+{
+    "id": 2,
+    "color": ""
+},
+{
+    "id": 3,
+    "color": {}
+},
+{
+    "id": 4
+}]
+```
+
+The filter `color IS EMPTY` selects the document ids `0`, `2` and `3` while the filter `color IS NOT EMPTY` or `NOT color IS EMPTY` selects the document ids `1` and `4`.
+
+The filter `color IS NULL` selects the document ids `1` while the filter `color IS NOT NULL` or `NOT color IS NULL` selects the document ids `0`, `2`, `3` and `4`.
+
+Done by @Kerollmops in #3571.
+
+## Improve relevancy and search performance
+
+TBD
```

Comment:
Search refactoring change should be done here @loiclec @dureuill 
For your information @Kerollmops and @gmourier